src/prefect/server/models/concurrency_limits_v2.py

```diff
@@ -188,22 +188,22 @@ async def delete_concurrency_limit(
 async def bulk_read_or_create_concurrency_limits(
     session: AsyncSession,
     names: List[str],
+    create_if_missing: Optional[bool] = True,
 ) -> List[orm_models.ConcurrencyLimitV2]:
     # Get all existing concurrency limits in `names`.
     existing_query = sa.select(orm_models.ConcurrencyLimitV2).where(
         orm_models.ConcurrencyLimitV2.name.in_(names)
     )
     existing_limits = list((await session.execute(existing_query)).scalars().all())
 
-    # If any limits aren't present in the database, create them as inactive.
+    # If any limits aren't present in the database, create them as inactive,
+    # unless we've been told not to.
     missing_names = set(names) - {str(limit.name) for limit in existing_limits}
 
-    if missing_names:
+    if missing_names and create_if_missing:
```

Comment:
Hmmm I think I get the idea of this `create_if_missing` flag but I'm not convinced it's additive - it's very specific to the `limit=1` case which is more like a lock than a rate limit or anything else.

Reply:
I can see why this would look bizarre. I don't believe it's specific to `limit=1`. I'll see if I can defend it.

I've wanted to add this option to the concurrency API for a long time, after getting multiple user complaints that global concurrency examples in our docs didn't work. E.g.: https://github.com/PrefectHQ/prefect/issues/11787.

If you use any of our global concurrency limit examples in the docs, it's easy to write code that uses a limit, run the code, and think the limits "didn't work." That's because we create them implicitly, but we make them inactive. So you have to go to the UI to activate the limit you just created in code. This remains true today.

This behavior of implicitly creating limits but making them inactive is wise, perhaps, but encountering it in practice feels wrong: I shouldn't have to leave my text editor to run a UI command or even a CLI command if I can otherwise control everything about a concurrency limit with the actual interface (`with concurrency(...)`.

So that's the origin story for `create_if_missing`. Why is it here, in this PR, though? It's only here because I mistakenly thought we should create a limit for the tag-based concurrency use case if a limit didn't exist, but that's obviously wrong because only some tags will be limits. 😂 In fact, we want to make sure don't create these limits if they don't already exist.

So you see, I added this option and then retained it for the negative case because I've wanted the `create_if_missing` option for so long.

A couple of possibilities:
1. You might disagree that `with concurrency(...)` should have an option to create a limit if it doesn't exist. Instead, users should visit the UI or CLI to activate an implicitly created flag. In this case, we should allow users to activate global concurrency flags via the CLI. (We should probably already do this, it's just not super high priority.)
2. You might agree that this option should exist but want it in a separate PR. That's fair. Though I would love to just ship it here, of course. 😀 
3. You might agree that this option should exist and it's fine to keep it in this PR.

Reply:
Wait, I may be conflating two different things. Stand by. 😂